### PR TITLE
Badges

### PR DESCRIPTION
**Pull request by aatishpandey**
image
includes:
- Badge on icons
- Badge on avatar

### CODE REVIEW

File: components/component-stylesheets/badges.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    height: 8rem;
+}
+
+.status{
+    border-radius: 50%;
+    background-color: var(--mr-primary-color);
+    height: 1.5rem;
+    width: 1.5rem;
+    position : absolute;
+    border: 0.3rem solid white;
```
Comment (shuntzhere) [inline components/component-stylesheets/badges.css]: You can use this white as a variable.

**Review by shuntzhere — APPROVED** [review]
Suggested a minor change, rest looks good.

**Comment by sadathshariff** [comment]
You can increase the size of icons a bit, everything else looks good

**Review by kjais1720 — APPROVED** [review]
Looks good👍

File: components/component-stylesheets/badges.css (hunk 1 above)
Comment (aatishpandey) [reply components/component-stylesheets/badges.css]: thanks for pointing it out. fixed in recent commit.

**Pull request merged**